packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -75,8 +75,64 @@ const defaultThemeColors = {
   },
 };
 
-export const defaultAppTheme: AppTheme = {
-  colors: defaultThemeColors,
+const darkThemeColors = {
+  beta: {
+    "100": "#1E1B26",
+    "200": "#23212D",
+    "400": "#946314",
+    "500": "#8A5D0C",
+    "600": "#7F5C1E",
+  },
+  error: {
+    "100": "#5E1313",
+    "200": "#612626",
+    "400": "#FF5468",
+    "500": "#E84551",
+    "600": "#FF2D46",
+  },
+  gray: {
+    "100": "#121212",
+    "200": "#1F1F1F",
+    "300": "#3C3C3C",
+    "400": "#666666",
+    "500": "#8A8A8A",
+    "600": "#AAAAAA",
+  },
+  info: {
+    "100": "#1A1F29",
+    "200": "#1C1F29",
+    "400": "#6792FF",
+    "500": "#4A70A5",
+    "600": "#396388",
+  },
+  primary: {
+    "100": "#181A23",
+    "200": "#1A1F29",
+    "300": "#5A6F92",
+    "400": "#6792FF",
+    "500": "#4A70A5",
+    "600": "#396388",
+  },
+  secondary: {
+    "400": "#6D8E3A",
+  },
+  success: {
+    "100": "#193A26",
+    "200": "#197C5F",
+    "400": "#72B07E",
+    "500": "#5F8F6B",
+    "600": "#345B3E",
+  },
+  warning: {
+    "100": "#302B23",
+    "200": "#382E24",
+    "400": "#F5A623",
+    "500": "#D8921F",
+    "600": "#946314",
+  },
+};
+
+export const sharedAppTheme: Omit<AppTheme, "colors" | "mode"> = {
```

Comment:
Not sure if I like this typing approach yet... At the very least explain the theme building system in comments here.

Reply:
Explained!